soh/soh/OTRGlobals.cpp

```diff
@@ -731,7 +731,7 @@ extern "C" SoundFontSample* ResourceMgr_LoadAudioSample(const char* path)
         sampleC->unk_bit26 = sample->unk_bit26;
         sampleC->unk_bit25 = sample->unk_bit25;
 
-        sampleC->book = new AdpcmBook[sample->book.books.size() * sizeof(int16_t)];
+        sampleC->book = new AdpcmBook[sizeof(AdpcmBook) + sample->book.books.size() * sizeof(int16_t)];
```

Comment:
Doesn't this change the semantics of the array value being looked up?

Reply:
no, this only changes the size of the allocation, but i see what i did was wrong, if books.size() is 0, then it should just be special cased. what i did was just wrong.